### PR TITLE
Json connection

### PR DESCRIPTION
**Pull request by claudiapao10**
Files Added:
* data.json - Sofia's file with the information of the word.

Files Changed:
* info.html - Added some elements as placeholders.
* information-display.js - Added Sofia's function that read the words of the json file and merged the with some alredy created functions.
* popup.html - Deleted 'example' radio button.
* word-selection-check.js - Deleted the 'example' section on the if statement.

Things to work on:
* Upon team discussion.

UI example:
ui1png
ui2

### CODE REVIEW

**Review by lolitsgab — APPROVED** [review]

**Review by GoogleyGabriel — APPROVED** [review]

File: project/chrome-extension/information-display.js [hunk 1]
```diff
@@ -1,35 +1,86 @@
...
-  const source = document.getElementById("sources");
+function addSources(sourceURL) {
   // TODO(@SamRod33 or @SofiaPSU): Add to the sources by using source.innerText
+
+      var a = document.createElement('a');
+      a.setAttribute('href',sourceURL);
+      // a.innerHTML = "Source: "+topics[i].source.sourceName;
+      a.innerHTML = "Source: "+ sourceURL;
+      document.getElementById('sources').appendChild(a);
+
```
Comment (SamRod33) [inline project/chrome-extension/information-display.js:45]: This uses the old version where urls was just an array of strings that were each link. See the my pull request on the branch json-update for how the urls structure has changed

File: project/chrome-extension/information-display.js [hunk 2]
```diff
@@ -1,35 +1,86 @@
...
 }
+  function loadJSON(callback) {
+    var xObj = new XMLHttpRequest();
+    xObj.overrideMimeType("application/json");
+    xObj.open('GET', 'data.json', true);
```
Comment (SamRod33) [inline project/chrome-extension/information-display.js:50]: Json file is called words.json, I know data.json is the example file, but we should integrate into using words.json

File: project/chrome-extension/information-display.js [hunk 3]
```diff
@@ -1,35 +1,86 @@
...
+      addPicture(json[word].image);
+      createDescription(json[word].description);
+      addSources(json[word].URL);
+    
+      // var json = JSON.parse(response);
+      // console.log('json response=', json);
+      //console.log('your local JSON =', JSON.stringify(json, null, 4));
+      // 4. render to your page
+      // const app = document.querySelector('#app');
+      // app.innerHTML = '<pre>' + json[word].description +'<br>' +json[word].image +'<br>'+ json[word].URL + '</pre>';
```
Comment (SamRod33) [inline project/chrome-extension/information-display.js:77]: Not sure if this is old code? This could be cleaned up if this is no longer being used

**Review by SamRod33 — APPROVED** [review]

**Review by SofiaPSU — APPROVED** [review]
Looks great! I think Sam already touched on the only issues I saw.

File: project/chrome-extension/information-display.js (hunk 3 above)
Comment (sebagabs) [inline project/chrome-extension/information-display.js:77]: Deleted commented code

**Review by sebagabs — APPROVED** [review]
Looks pretty good. We gotta make sure to implement the lastest JSON file changes, and note that there should be a space after the // in comments: `// comment`

**Pull request merged**